### PR TITLE
Classic gray theme: Show categories in product detail breadcrumb

### PR DESCRIPTION
**Pull request by tulimaki**
Refs SHOOP-1386

### CODE REVIEW

File: shoop/themes/classic_gray/templates/classic_gray/shoop/front/product/detail.jinja [hunk 1]
```diff
@@ -19,9 +19,14 @@
         <li><a href="/">{% trans %}Home{% endtrans %}</a></li>
         {% set category = shop_product.primary_category %}
         {% if category %}
-            {% for c in category.get_ancestors() if c.is_visible(customer=request.customer) %}
-                <li><a href="{{ shoop.urls.model_url(c) }}">{{ c.name }}</a></li>
+            {% for child_cat in category.get_ancestors() if child_cat.is_visible(customer=request.customer) %}
+                {% set child_cat_url = url("shoop:category", pk=child_cat.pk, slug=child_cat.slug) %}
```
Comment (akx) [inline shoop/themes/classic_gray/templates/classic_gray/shoop/front/product/detail.jinja]: Why doesn't this use `shoop.urls.model_url()`?

File: shoop/themes/classic_gray/templates/classic_gray/shoop/front/product/detail.jinja [hunk 2]
```diff
@@ -19,9 +19,14 @@
...
+                <li><a href="{{ child_cat_url }}">{{ child_cat.name }}</a></li>
             {% endfor %}
+            {% set category_url = url("shoop:category", pk=category.pk, slug=category.slug) %}
```
Comment (akx) [inline shoop/themes/classic_gray/templates/classic_gray/shoop/front/product/detail.jinja]: Why doesn't this use `shoop.urls.model_url()`?

File: shoop/themes/classic_gray/templates/classic_gray/shoop/front/product/detail.jinja [hunk 3]
```diff
@@ -19,9 +19,12 @@
         <li><a href="/">{% trans %}Home{% endtrans %}</a></li>
         {% set category = shop_product.primary_category %}
         {% if category %}
-            {% for c in category.get_ancestors() if c.is_visible(customer=request.customer) %}
-                <li><a href="{{ shoop.urls.model_url(c) }}">{{ c.name }}</a></li>
+            {% for child_cat in category.get_ancestors() if child_cat.is_visible(customer=request.customer) %}
```
Comment (akx) [inline shoop/themes/classic_gray/templates/classic_gray/shoop/front/product/detail.jinja]: These are parent cats, not children.

Comment (akx) [inline shoop/themes/classic_gray/templates/classic_gray/shoop/front/product/detail.jinja]: You can pass `include_self=True` to `get_ancestors` to simplify the loop further.

Comment (RauliL) [inline shoop/themes/classic_gray/templates/classic_gray/shoop/front/product/detail.jinja]: Parent cat:
16f89c17f152fe0c98380c94fef14a35

Comment (akx) [inline shoop/themes/classic_gray/templates/classic_gray/shoop/front/product/detail.jinja]: That's what I was going for. :smile_cat:

File: shoop/themes/classic_gray/templates/classic_gray/shoop/front/product/detail.jinja [hunk 4]
```diff
@@ -19,9 +19,11 @@
         <li><a href="/">{% trans %}Home{% endtrans %}</a></li>
         {% set category = shop_product.primary_category %}
         {% if category %}
-            {% for c in category.get_ancestors() if c.is_visible(customer=request.customer) %}
-                <li><a href="{{ shoop.urls.model_url(c) }}">{{ c.name }}</a></li>
+            {% for parent_cat in category.get_ancestors(include_self=True) if parent_cat.is_visible(customer=request.customer) %}
```
Comment (akx) [inline shoop/themes/classic_gray/templates/classic_gray/shoop/front/product/detail.jinja]: Actually, you'd get a nicer, shorter diff if you didn't rename `c` now :)

**Pull request merged**